src/utils/ledger.js

```diff
@@ -34,6 +34,38 @@ const formatPathData = (index) => {
   return buffer;
 };
 
+const intToByte = (number) => {
+  const buf = Buffer.alloc(1);
+  buf[0] = number;
+  return buf;
+}
+
+export const serializeTokenInfo = (token, hasSignature) => {
+  // version + uid + len(symbol) + symbol + len(name) + name
+  const uidBytes = Buffer.from(token.uid, "hex");
+  const symbolBytes = Buffer.from(token.symbol, "utf8");
+  const nameBytes = Buffer.from(token.name, "utf8");
+  const arr = [];
+
+  // 0: token version = 1 (always)
+  arr.push(intToByte(1));
```

Comment:
Maybe we should create this in the constants file

Reply:
Added